### ISSUE TITLE
Debian 8 Install Error

### ISSUE THREAD

**Issue by justinbowers** [sentora/sentora-installers#107]
Hi. I'm new to Sentora and had an issue when attempting to install it on a fresh VPS under my Linode account. I am running the latest version of Debian 8 with all patches installed.

When running the install command:

`bash <(curl -L -Ss http://sentora.org/install)`

I receive the following:

> Checking that minimal requirements are ok
> Detected : Linux  4.8.3-x86_64-linode76  x86_64
> Sorry, this OS is not supported by Sentora.

I saw a couple of install issues posted in reference to the same error for Debian 7, but it had appeared that the issues were resolved and closed. Any help is greatly appreciated.

**Comment by justinbowers** [comment]
As a note, I just tried running the latest master installer with the following command and it started the installation of Sentora successfully:

`bash <(curl -L -Ss https://raw.githubusercontent.com/sentora/sentora-installers/master/sentora_install.sh)`

I know it mentions to only run it for testing purposes and is for developers or project staff, so I was wondering when the install bug for the latest stable installer will be fixed. The installation did throw some errors when installing Logrotate referencing the following:

> -- Installing Logrotate
> ln: failed to create symbolic link ‘/etc/logrotate.d/Sentora-apache’: File exists
> ln: failed to create symbolic link ‘/etc/logrotate.d/Sentora-postifx’: File exists
> ln: failed to create symbolic link ‘/etc/logrotate.d/Sentora-dovecot’: File exists
> Failed to restart postfix.service: Unit postfix.service failed to load: No such file or directory.

I was able to login to Sentora fine after a reboot though. Thanks for any information or assistance.
